### DISCUSSION TITLE
Need help with "next export -o <dir>" cannot be used when "output: export"

### DISCUSSION THREAD

**Discussion by MiguelTrinca** [vercel/next.js#57443] [App Router] [answered]
Hi everyone!

I'm new to Next.js and Vercel (and javascript for that matter ahah) I'm trying to deploy my application to vercel but I keep incountering the following error:

`
(Static)  automatically rendered as static HTML (uses no initial props)
 "next export -o <dir>" cannot be used when "output: export" is configured in next.config.js. Instead add "distDir" in next.config.js https://nextjs.org/docs/advanced-features/static-html-export
`

My `next,config.js` is the following:

\/**
 /* @type {import('next').NextConfig}
 */
const nextConfig = {
  output: 'export',
}
 
module.exports = nextConfig

Would love to hear some feedback on how to solve this issue.

Additional Notes
--

- Currently using nvm 18
- Currently using next 13.2.4
- I've tried to change the `output: 'export'` to `distDir: 'dir'`

Also
---
I've search Github issues about this but it didn't help

- https://github.com/vercel/next.js/issues/48207
- https://github.com/vercel/next.js/pull/51191

**Comment by icyJoseph** [comment]
Hi,

So you are doing `next export` through a script in package.json or something like that right?

In the latest versions, if you have `output: "export"` in your next.config.js, then all you gotta do is `next build`, and that'll take care of also exporting it. To choose where output the export, use the `distDir` property in the next.config.js file as well.

**Reply by MiguelTrinca** [reply, marked as answer]
Hi!

Thank you for the response! With your help I was able to find the problem!

I don't know why, but in my _package.json_ I had the build script like this:
"build": "next build && next export -o build/"

The error indicated that I had to change _next.config.js_, but it was actually with _package.json_

All I did was removing the `&& next export -o build/` from _package.json_ and removed the `export: "output"` from _next.config.js_
